### ISSUE TITLE
Commercial SDK use

### ISSUE THREAD

**Issue by fotiDim** [spotify/ios-sdk#807]
I am fed up getting no answer from Spotify for commercial SDK use and decided to open an issue here to raise awareness. How do even developers justify carrying on with development using the Spotify SDK when they cannot monetise their apps? You are basically turning people away from your product and I really wonder why you even have an SDK if it just aimed at hobbyists. I would like to hear an official reply or workarounds from other developers regarding this very disturbing issue.

**Comment by danielr** [comment]
+1

It's understandable that Spotify doesn't want third-party apps to appear selling Spotify content itself, but prohibiting *any* monetization of apps that add further value to the streaming experience is defeating the purpose of an SDK. Limiting the potential adopters to "personal use" hobby apps will most likely not lead to many high-quality, long-term maintained apps, IMHO.

**Comment by fotiDim** [comment]
I don't want to sell or monetise on the content itself. I want to offer added value services and I target a very niche market which I seriously doubt that the Spotify app would be interested in. 

Also keep in mind that 3rd party apps require premium Spotify accounts so just by existing my app gives incentives to users to upgrade to Premium thus making money for Spotify. It seems unfair not even to be able to cover my development costs.

What they could at least do is loosen their commercial TOS and allow certain cases. And definitely be more responsive regarding that matter.

**Comment by escamoteur** [comment]
Yeah, me to am missing the logic behind this. I have no problems to show clearly that the App is a client for Spotify Premium Customers, so I don't understand their policy.

**Comment by NeoLegends** [comment]
I'm pretty sure this is a requirement from the content owners and the music labels. They probably fear that people would make big money (TM) by selling the music. They probably do not think like developers, so they probably don't see the issues we are facing.

**Comment by escamoteur** [comment]
Sorry but that does not make any sense. Spotify handles DRM so there is no threat

**Comment by edwinfinch** [comment]
You guys make some great points. 

We applied for Spotify commercialization approval back at the beginning of July in 2016, and got a reply 3 months later rejecting us which luckily outlined the reasons why. We fixed those reasons and reapplied in December, and still are waiting to hear back. I wish we had reapplied sooner, realizing now every wait for application will probably be 3 months or more.

We expect a reply in March, and honestly, I don't expect to be approved. I'm truly sorry to say that I am quite pessimistic about the chances of getting approved, because it seems they'd rather go through your app with a fine tooth comb to find something wrong with it, than actually help you succeed in being approved for commercialization.

As well, since Spotify's developer team is ignoring this, I assume it's something edgy/touchy internally at Spotify that they are trying to avoid conversation of.

It's a shame, honestly. Apple Music encourages developers to use their platform (ie. they even have an affiliate program), though Spotify is overall the superior service so it leaves you stuck between two worlds as a music streaming app developer.

**Comment by fotiDim** [comment]
@edwinfinch at least you got a reply back. I have applied 3 times within 2 years time (using the old and the new form) and never got a reply back. I decided to ditch Spotify for my app and turn to alternatives.
